edge-modules/TestAnalyzer/DeviceAnalysis.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestAnalyzer
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Serialization;
+
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    class DeviceAnalysis
```

Comment:
this name doesn't tell me anything about test result?
```suggestion
    class TestResultAnalysis
```

Reply:
Current implementation require changes for any new set of test result.  we can discuss in person.